content/MLaaS4HEP.md

```diff
@@ -0,0 +1,94 @@
+## Machine Learning as a Service for HEP
+
+MLaaS for HEP is a set of Python based modules to support reading HEP data and
+stream them to ML of user choice for training. It consists of three independent layers:
+- Data Streaming layer to handle remote data, see [reader.py](https://github.com/vkuznet/MLaaS4HEP/blob/master/src/python/MLaaS4HEP/reader.py)
+- Data Training layer to train ML model for given HEP data, see [workflow.py](https://github.com/vkuznet/MLaaS4HEP/blob/master/src/python/MLaaS4HEP/workflow.py)
+- Data Inference layer, see [tfaas_client.py](https://github.com/vkuznet/TFaaS/blob/master/src/python/tfaas_client.py)
+
+The MLaaS4HEP resopitory can be found [here](https://github.com/vkuznet/MLaaS4HEP).
+
+The general architecture of MLaaS4HEP looks like this:
+![MLaaS4HEP-architecture](https://github.com/vkuznet/MLaaS4HEP/blob/master/images/MLaaS4HEP_arch_gen.png)
+
+Even though this architecture was originally developed for dealing with
+HEP ROOT files we extend it to other data formats. So far the following
+data formats are supported: JSON, CSV, Parquet, ROOT. The former ones support

```

Comment:
"..., and ROOT."

Instead of, "The former ones...," how about, "All of the formats support reading files from the local file system or HDFS, while only the ROOT format supports reading files via the [XRootD](https://xrootd.slac.stanford.edu/) protocol."